rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -870,12 +875,13 @@ public boolean executeCode(
 
         returnDataBuffer = null; // reset return buffer right before the call
         ProgramResult childResult = null;
+
         ProgramInvoke programInvoke = programInvokeFactory.createProgramInvoke(
                 this, new DataWord(contextAddress.getBytes()),
                 msg.getType() == MsgType.DELEGATECALL ? getCallerAddress() : getOwnerAddress(),
                 msg.getType() == MsgType.DELEGATECALL ? getCallValue() : msg.getEndowment(),
-                limitToMaxLong(msg.getGas()), contextBalance, data, track, this.invoke.getBlockStore(), byTestingSuite());
-
+                limitToMaxLong(msg.getGas()), contextBalance, data, track, this.invoke.getBlockStore(),
+                msg.getType() == MsgType.STATICCALL || isStaticCall(), byTestingSuite());
```

Comment:
indent this line (884) so it doesn't look like a separate statement

Reply:
It's fine, is indented as the previous one because it has the same weight, both are part of the parameters of `createProgramInvoke`

Reply:
This is an argument to `limitToMaxLong`, so it should be indented... anyway, not important.